tokenomics-explainers/002-rework-intro.md

```diff
@@ -44,10 +49,21 @@ Node-level penalties and forced validator exits are also included in the rework
 
 Megapools will enable multiple validators to share the same Ethereum withdrawal address – this **reduces initial and ongoing gas costs** for node operators.
 
-**RPL inflation will be reduced from 5% to 1.5%**, because the current 3.5% inflation used for RPL rewards to NOs is no longer necessary.
+**RPL issuance will be reduced from 5% to 1.5%**, because the current 3.5% issuance used for RPL rewards to NOs is no longer necessary.
 
 Forced delegate upgrades (with a grace period) will limit the technical debt the Rocket Pool protocol accumulates from previous versions.
 
+## Personas
+
+|Persona|Now|With Rework|
+|-|-|-|
+|**Existing Rocket Pool Node Operator**|I need RPL to create minipool validators. I have to remain collateralized at an RPL staked value of at least 10% of my borrowed ETH to receive RPL issuance rewards. My commission is 14% of my borrowed ETH.|I do not need RPL to create megapool validators. My commission is 3.5% of my borrowed ETH. If I choose to stake RPL I will receive additional ETH rewards (5% of protocol revenue from all megapools) proportional to my share of staked RPL (up to a maximum). I do not need to maintain a minimum stake to receive these rewards. I will benefit from value accrual to RPL (5.5% of protocol revenue from all megapools) or a 5.5% increase to the rewards for staking RPL. I will no longer receive rewards in RPL.
+|**rETH holder**|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Currently my rewards are ~86% of solo staking APY. The fee is paid to NOs as commission for their services.|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Initially my rewards are unchanged at ~86% of solo staking APY. The fee is considered revenue for Rocket Pool and will be split by the pDAO to several targets. The total fee may be modified by future pDAO votes.|
```

Comment:
```suggestion
|**rETH holder**|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Currently my rewards are ~86% of solo staking APY since I am charged a fee of ~14%.|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Currently my rewards are ~86% of solo staking APY since I am still charged a total fee of ~14%. The total fee may be modified by future pDAO votes.|
```

Reply:
If we want to really zoom in on rETH holders, they don't care what we do with the fee...